README.md

```diff
@@ -394,8 +413,10 @@ will be available (provided that `iex` is started
 iex> Math.sum(1, 2)
 3
 ```
+***
 
-## Generating your first Elixir project
+
+## *Generating your first Elixir project*
```

Comment:
Is this meant to italicise this title? Or did you mean to make it a 3 hash title (given no other titles are in italics)?